assets/src/components/Lightbox/setupLightboxForImageBlocks.js

```diff
@@ -0,0 +1,34 @@
+import { Lightbox } from './Lightbox';
+
+export const setupLightboxForImageBlocks = function() {
+    // We can't use `createPortal` outside `render()`,
+    // `React.render()` needs a node, even if it ends up being empty.
+    // See: https://github.com/facebook/react/issues/12653#issuecomment-382851495
+    const lightBoxNode = document.createElement('div');
+    lightBoxNode.style.position = 'fixed';
+  
+    document.body.appendChild(lightBoxNode);
+  
+    const imageBlocks = document.querySelectorAll('.wp-block-image');
```

Comment:
Maybe we can add something like `:not(.force-no-lightbox)` to this selector, as a quick way to implement excluding images by adding a class in the sidebar?

Reply:
Nice, added it. 👍